src/stores/security-two-factor-authentication-use.md

```diff
@@ -107,66 +155,20 @@ The Authy authenticator includes multiple types of apps available from this page
 
-## Duo
-
-The Duo Mobile app is available through [Google Play][3] or [iOS App Store][4]. To use, it must be activated and linked to an account. For details, your security staff should obtain a Duo account and complete setup.
+## U2F (Yubikey and others)
 
-### Step 1: Configure Duo
+With the U2F device configured, the user attaches the device to their system when logging into the _Admin_. For more information, check your U2F vendor documentation, such as [YubiKey][1] by Yubico.
 
-1. Enter Magento Admin account credentials and sign in to your account.
-
-1. When the Duo Setup page appears. Click **Start setup** and do the following:
-
-    ![Example storefront - Duo setup]({% link images/images/storefront-2fa-duo-user1.png %}){: .zoom}
-    _Duo Setup_
-
-   - Select your device.
+1. Enter Magento _Admin_ account credentials and sign in to your account.
 
-        ![Duo authentication - select device ]({% link images/images/storefront-2fa-duo-user2.png %}){: .zoom}
-        _Device Type_
-
-   - When prompted, enter your **phone number**, and click **Continue**.
-
-        This example requests your phone number, because we are using a mobile device.
-
-        ![Duo authentication - enter phone number]({% link images/images/storefront-2fa-duo-user3.png %}){: .zoom}
-        _Enter Your Phone Number_
-
-   - When prompted to install Duo Mobile for your phone type, click <span class="btn">I have Duo Mobile</span>.
-
-        ![Duo authentication - verify app installation]({% link images/images/storefront-2fa-duo-user5.png %}){: .zoom}
-        _Verify App Installation_
-
-1. Open Duo Mobile, and scan the **QR code** to sync the authenticator with Magento. A checkmark appears when the activation is complete.
-
-    ![Duo authentication - verification code]({% link images/images/storefront-2fa-duo-user6.png %}){: .zoom}
-    _Duo Verification Code_
-
-1. To configure your settings for the device, choose the action that you want to take place when you sign in.
-
-   - `Ask me to choose an authenticator method` — Allows the user to select when logging in and authenticating in the Magento Admin.
-   - `Automatically send this device a Duo Push` — Sends a message to your device to accept or deny for access.
-   - `Automatically call this device` — Calls and provides a passcode for entering
-
-    ![Duo verification actions]({% link images/images/storefront-2fa-duo-user7.png %}){: .zoom}
-    _Duo verification code_
-
-### Step 2: Sign In with Duo
-
-The following example displays the options when selecting Ask me to choose an authenticator method:
-
-1. When prompted, enter Enter Magento Admin credentials to log in.
-
-    ![Duo - signin]({% link images/images/storefront-2fa-duo-auth.png %}){: .zoom}
-    _Duo access_
+    ![U2F key access]({% link images/images/storefront-2fa-u2f.png %}){: .zoom}
+    _U2F Key Access_
 
-1. Choose the method that you want to use to authenticate:
+1. Press the button on the key.
```

Comment:
is this in the Admin?   should it be click?

Reply:
No, it's a literal button on a device.